network/load-balancer/reference-content/configuring-backends.mdx

```diff
@@ -0,0 +1,132 @@
+
+## Configuring traffic management
+
+### Balancing method
+
+Load Balancers support three different modes of balancing load (requests) between backend servers:
+
+* **Round-robin**: Requests are sent to each backend server in turn, with the Load Balancer acting like a turnstile. This is the most frequently used balancing method, and the easiest to implement. It is well-suited to infrastructures that have identical backend servers.
+* **Least connections**: Each request is assigned to the server with the fewest active connections. This method works best when it is expected that sessions will be long, e.g. LDAP, SQL TSE. It is less well suited to protocols with typically short sessions like HTTP.
+* **First available** : Each request is directed towards the first backend server with [available connection slots](#backend-protection). Once a server reaches its limit of maximum simultaneous connections, requests are directed to the next server. This method uses the smallest number of servers at any given time, which can be useful if, for example, you sometimes want to power off extra servers during off-peak hours.
+
+### Server IPs (backend servers)
+
+You can add one or more IP addresses, either IPv4 or IPv6, of the backend servers your Load Balancer's backend will forward traffic to.
+
+### Sticky sessions
+
+When activated, sticky sessions bind a user's session to a specific server in the pool of backend servers. This ensures that all subsequent sessions from the user are sent to the same backend server while there is at least one active session. Sticky sessions can be **cookie based** or **IP based** (aka table-based).
+
+- **IP-based** stickiness uses the source (client) IP address to stick sessions to backends. This is the only possible method for TCP backends.
+- **Cookie-based** stickiness uses an HTTP cookie to identify a session, and the server in the backend pool that it will stick to. For each incoming request without a stickiness cookie, the Load Balancer creates a cookie and selects a backend for the session. Subsequent requests providing this cookie will land on the same backend. You can customize the name of the cookie.
+
+## Configuring advanced settings
+
+Recommended settings for all the parameters below are selected by default, however you can modify them if you wish.
+
+### Timeout
+
+Timeout settings allow you to define the maximum time that backend servers should be given to establish connections and to process requests. The following timeout settings are available:
+
+- **Tunnel timeout**: Defines the maximum allowed tunnel inactivity time (in ms) between a client and a backend server after a Websocket is established. This value takes precedence over client and server timeout in determining when to close the connection. The default value is 900 000, the minimum value is 0 and the maximum value is 2 147 483 647.
```

Comment:
@RoRoJ No need to specify the max value it's not really relevant given how big it is. Default and min are interesting.

Reply:
I see your point but I would rather leave the max value there so that at least the documentation is complete, rather than leave the reader wondering if we forgot to include it.